router/router.go

```diff
@@ -77,6 +91,14 @@ func (r *router) handleRoutes() {
 	// GET /healthz - returns a JSON health check response indicating the ready status of PATH
 	r.mux.HandleFunc("GET /healthz", methodCheckMiddleware(r.healthChecker.HealthzHandler))
 
+	// GET /v1/sanctioned_endpoints - returns a JSON list of sanctioned endpoints
```

Comment:
Do we maintain an OpenAPI schema (and in turn, swagger UI) anywhere?

If not, can you create a TODO_IMPROVE?

Will be very valuable very soon.